src/test/kotlin/org/rust/lang/core/completion/RsFragmentSpecifierCompletionTest.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.completion
+
+import org.intellij.lang.annotations.Language
+import org.rust.lang.core.macros.FragmentKind
+
+class RsFragmentSpecifierCompletionTest : RsCompletionTestBase() {
+    fun `test fragment specifiers`() = checkContains("""
+        macro_rules! foo {
+            ($ l:/*caret*/) => {}
+        }
+    """)
+
+    fun `test fragment specifiers after name`() = checkNotContains("""
+        macro_rules! foo {
+            ($ i/*caret*/) => {}
+        }
+    """)
+
+    fun `test fragment specifiers after $ sign`() = checkNotContains("""
+        macro_rules! foo {
+            ($/*caret*/) => {}
+        }
+    """)
+
+    @Suppress("SameParameterValue")
+    private fun checkContains(@Language("Rust") text: String) {
+        for (fragmentKind in FragmentKind.kinds) {
```

Comment:
Oh. Is it not possible to implement it without such loops? With it, you invoke parsing and completion `FragmentKind.kinds.size` times instead of one.

(I know there is the same code in `RsPartialMacroArgumentCompletionTest`, but there are a very few numbers of elements, mostly 1)

Reply:
I referred to:
https://github.com/intellij-rust/intellij-rust/blob/d640c47d97ac9bbad2cb1d3876e74104a38e9a3c/src/test/kotlin/org/rust/lang/core/completion/RsPrimitiveTypeCompletionTest.kt#L23-L28

Reply:
Should I fix?

Reply:
@t-kameyama OK, I think we can merge it as is and try to fix both tests later!